datalad/metadata/search.py

```diff
@@ -173,16 +175,48 @@ def _deep_kv(basekey, dct):
             else:
                 yield key, v
 
+    def get_indexer_for(metadata_format_name: str) -> callable:
+        indexer = (tuple(iter_entry_points('datalad.metadata.indexers', metadata_format_name)) or (None,))[0]
+        if isinstance(indexer, EntryPoint):
+            try:
+                indexer_object = indexer.load()(metadata_format_name)
+                return indexer_object.create_index
+            except Exception as e:
+                lgr.warning('Failed to load indexer %s: %s', indexer.name, exc_str(e))
```

Comment:
We assume that at most one indexer with the given name exists. Everything else should be an error since we have not provided a way to discriminate between different indexers for the same extractor-output.

Reply:
ok, let's just log about that, added comment above, marking this resolved